content/millennium/dstu2/individuals/patient.md

```diff
@@ -71,11 +75,28 @@ Notes:
 - The `birthdate` parameter may be provided twice to indicate a date range, but must contain the inclusive prefixes 'le' and 'ge'
 - The `birthdate` parameter may be provided once with the following prefixes: 'ge', 'le', 'gt', 'lt', 'eq'
 
-### Response
+### Headers
```

Comment:
+1 to removing comment text and linking to other documentation for media type and auth headers